### PR TITLE
Fix and delete invalid links

### LINKED ISSUE #140: Sweep for dead links
In reviewing https://github.com/kubernetes/community/pull/2408, I noticed that http://cherrypick.k8s.io/#/queue times out. We should sweep the nginx config for dead links and remove them.

/sig contributor-experience
/help

### PR DESCRIPTION
**Pull request by nzoueidi**
Fixing and deleting invalid links in readme file. This PR close #140 

/sig contributor-experience

### CODE REVIEW

File: k8s.io/README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Vanity URL(s)
 |  | k8s.io | kubernetes.io |
 | --- | --- | --- |
 | APT downloads| https://apt.k8s.io | https://apt.kubernetes.io |
-| Blog | https://blog.k8s.io | |
+| Blog | https://blog.k8s.io | https://blog.kubernetes.io |
```
Comment (cblecker) [inline k8s.io/README.md]: This should be https://kubernetes.io/blog/

File: k8s.io/README.md [hunk 2]
```diff
@@ -20,14 +20,13 @@ Vanity URL(s)
...
-| Redirections | https://go.k8s.io | https://go.kubernetes.io |
 | Issues | https://issue.k8s.io <br> https://issues.k8s.io | https://issue.kubernetes.io <br> https://issues.kubernetes.io |
-| Main site | https://k8s.io |
+| Main site | https://k8s.io | https://kubernetes.io |
 | PR Dashboard | https://pr-test.k8s.io | https://pr-test.kubernetes.io |
 | Pull requests | https://pr.k8s.io <br> https://prs.k8s.io | https://pr.kubernetes.io <br> https://prs.kubernetes.io |
 | Downloads | https://releases.k8s.io <br> https://rel.k8s.io | https://releases.kubernetes.io <br> https://rel.kubernetes.io |
 | Kubernetes SIGs | https://sigs.k8s.io | |
-| Tide status (formerly submit queue) | https://submit-queue.k8s.io | https://kubernetes.submit-queue.k8s.io |
+| Tide status (formerly submit queue) | https://submit-queue.k8s.io | https://submit-queue.kubernetes.io |
```
Comment (cblecker) [inline k8s.io/README.md]: This should be https://prow.k8s.io/tide

File: k8s.io/README.md [hunk 3]
```diff
@@ -20,14 +20,13 @@ Vanity URL(s)
 | Features repo | https://feature.k8s.io <br> https://features.k8s.io |  https://feature.kubernetes.io <br> https://features.kubernetes.io |
 | Install script | https://get.k8s.io | https://get.kubernetes.io |
 | Github organization| https://git.k8s.io | https://git.kubernetes.io |
-| Redirections | https://go.k8s.io | https://go.kubernetes.io |
```
Comment (cblecker) [inline k8s.io/README.md]: Why is this deleted?

File: k8s.io/README.md (hunk 1 above)
Comment (nzoueidi) [reply k8s.io/README.md]: That's true! then also blog.k8s.io need to k8s.io/blog. right?

File: k8s.io/README.md (hunk 3 above)
Comment (nzoueidi) [reply k8s.io/README.md]: As far as I can see here it is an invalid url. it returns 404 not found.

Comment (cblecker) [reply k8s.io/README.md]: It's live. The config for it is here: https://github.com/kubernetes/k8s.io/blob/a7985e6b571de579941b2a78135a3035f7951d92/k8s.io/configmap-nginx.yaml#L246-L269

It's a redirector, so the / URL doesn't do anything.

File: k8s.io/README.md (hunk 1 above)
Comment (cblecker) [reply k8s.io/README.md]: yes

File: k8s.io/README.md (hunk 3 above)
Comment (nzoueidi) [reply k8s.io/README.md]: Got it, thanks. I will make a commit including all the required changes.

File: k8s.io/README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@ Vanity URL(s)
 |  | k8s.io | kubernetes.io |
 | --- | --- | --- |
 | APT downloads| https://apt.k8s.io | https://apt.kubernetes.io |
-| Blog | https://blog.k8s.io | |
```
Comment (ixdy) [inline k8s.io/README.md]: This is the wrong fix - we should fix the actual issue by making `https://blog.k8s.io/` go to the right place, i.e. update the nginx config in `k8s.io/configmap-nginx.yaml`.
